### DISCUSSION TITLE
Inpainting model for lstein/invoke-ai

### DISCUSSION THREAD

**Discussion by dinandmentink** [AbdBarho/stable-diffusion-webui-docker#217] [Ideas]
lstein/invoke-ai support using the runway inpainting model. Would you be open to a pull request introducing the inpainting model?

This would entail:

- adding a download of the inpainting model
- changing the lstein models.yaml to allow loading the inpainting model (`default: false`)

Links:

https://invoke-ai.github.io/InvokeAI/installation/INSTALLING_MODELS/
https://huggingface.co/runwayml/stable-diffusion-inpainting

**Comment by AbdBarho** [comment]
@dinandmentink that would be great! I had it on my todo list from #203.

One thing to keep in mind is the ability for the model to be used by the different UIs, and if possible, a download that does not entail using a huggingface token, as it would make the setup process much more convoluted.

**Reply by AbdBarho** [reply]
implemented in #221
